Documentation/CRDs/ceph-nfs-crd.md

```diff
@@ -114,6 +115,10 @@ The `security` spec sets security configuration for the NFS cluster.
     determine the full service principal name: `<principalName>/<namespace>-<name>@<realm>`.
     e.g., nfs/rook-ceph-my-nfs@example.net. For full details, see the
     [NFS security doc](../Storage-Configuration/NFS/nfs-security.md#nfs-service-principals).
+  * `domainName`: this is the domain name used in the kerberos credentials. This is used to
+    configure idmap to map the kerberos credentials to uid/gid.
+    eg., DOMAIN1.EXAMPLE.COM.
+    [NFS security doc](../Storage-Configuration/NFS/nfs-security.md#kerberos-domain-name).
```

Comment:
It seems good to explain to the user what would happen if this is left empty, if that is behavior they might want. For me, I don't fully understand what the impacts of `NFS Ganesha will use the configured anonuid/anongid when accessing the local filesystem` means. Do you think it's useful to explain how this impacts NFS clients? Despite my experience with Rook, I may have less understanding of NFS server-client behavior and common knowledge than the average user of the feature.

```suggestion
  * `domainName`: the domain name used by idmap to map the kerberos credential to the
    user uid/gid. Without this configured, NFS Ganesha will use the configured
    anonuid/anongid when accessing the local filesystem.
    eg., DOMAIN1.EXAMPLE.COM.
    [NFS security doc](../Storage-Configuration/NFS/nfs-security.md#kerberos-domain-name).
```